eng/helixcorefxtests.proj

```diff
@@ -0,0 +1,40 @@
+<Project>
+  <Import Sdk="Microsoft.DotNet.Helix.Sdk" Project="Sdk.props" />
+
+  <PropertyGroup>
+    <HelixTargetQueue>''</HelixTargetQueue>
+    <TestAssetBlobFeedUrl>https://dotnetfeed.blob.core.windows.net/dotnet-core</TestAssetBlobFeedUrl>
+    <MicrosoftPrivateCoreFxNETCoreAppVersion>4.6.0-preview8.19353.6</MicrosoftPrivateCoreFxNETCoreAppVersion>
+    <__BuildOS>OSX</__BuildOS>
+    <__BuildArch>x64</__BuildArch>
+    <_TargetGroup>netcoreapp</_TargetGroup>
+  </PropertyGroup>
+
+  <!-- Import Microsoft.DotNet.Build.Tasks.Feed.targets for `ParseBuildManifest` -->
+  <Import Project="/Users/egorb/.nuget/packages/microsoft.dotnet.build.tasks.feed/2.2.0-beta.19356.1/build/Microsoft.DotNet.Build.Tasks.Feed.targets" />
```

Comment:
If you planned to check the CI you may need to update this path ;-)

Reply:
nah this PR is more like a sample -- it's too many helix variables I need to set and I have no idea how